src/lib/components/Textfield.react.js

```diff
@@ -0,0 +1,139 @@
+// @flow
+
+import React, { Component } from 'react';
+
+import { default as MuiTextField } from '@material-ui/core/TextField';
+import lightBaseTheme from 'material-ui/styles/baseThemes/lightBaseTheme';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+
+type Props = {
+  /** If true, the input element will be focused during the first mount */
+  autoFocus?: boolean,
+  /** The classes to be applied to this component. This keys in this object must be valid CSS rule
+   * names, and the values must be strings for the classnames to be assigned to each rule name
+   * Valid rule names are:
+    root
+    formControl
+    focused
+    disabled
+    colorSecondary
+    underline
+    error
+    multiline
+    fullWidth
+    input
+    inputMultiline
+   */
+  classes?: {
+    root?: string,
+    formControl?: string,
+    focused?: string,
+    disabled?: string,
+    colorSecondary?: string,
+    underline?: string,
+    error?: string,
+    multiline?: string,
+    fullWidth?: string,
+    input?: string,
+    inputMultiline?: string,
+  },
+  /** CSS class name of the root element */
+  className?: string,
+  /** 'primary' | 'secondary' */
+  color: string,
```

Comment:
Would you make this an optional prop?

Reply:
yes, it need to be declared as `color?: string` right?

Reply:
Yes, that's it.